Obtener tiempos de lectura de JSON en Python

MarcoOrtegaF opened the pull request:
Fixes #29

- Ahora el ECATEPEC.json esta en la carpeta assets

- El notebook de TemposLectura esta actualizado con graficas y el benchmarking en C++

MarcoOrtegaF committed:
Obtener tiempos de lectura de JSON en Python
Fixes #29

edoomm reviewed (changes requested):
Elimina el GeoJSON de `examples` y poner las funciones de `plot_time_vs_step_size` y demás en `plotfn.py`

MarcoOrtegaF committed:
- Se elimino el ECATEPEC.json de
    la carpeta examples
- La funcion plot_time_vs_step_size ahora esta
    disponible en el plotfn.py

MarcoOrtegaF committed:
Merge branch 'main' into MarcoOrtegaF/issue29

MarcoOrtegaF committed:
Update plotfn.py

Modificado a mano para que funcione bien

MarcoOrtegaF committed:
Update plotfn.py pt2

Ya mande alv la funcion "plot_time_vs_step_size"

MarcoOrtegaF committed:
Ahora esta disponible en assets la carpeta de "JSON2" con diversos txt y JSON

edoomm reviewed (changes requested):
Hmmm ya son muchos GeoJSON y no sé si nos sea conveniente tener todo esto aquí en este repositorio, ni los estamos usando y no creo los vayamos a usar todos ¿? No sé que opinen @JavierMung @MarcoOrtegaF yo preferiría que creemos otro repositorio aparte para todos estos assets

MarcoOrtegaF commented:
> Hmmm ya son muchos GeoJSON y no sé si nos sea conveniente tener todo esto aquí en este repositorio, ni los estamos usando y no creo los vayamos a usar todos ¿? No sé que opinen @JavierMung @MarcoOrtegaF yo preferiría que creemos otro repositorio aparte para todos estos assets

Pero igual no estorban en este repo ?no?
Siento que si los usamos vamos a tener que andar cambiando entre diferentes repos, pero igual yo diria que le preguntemos al doc Amilcar para que nos diga que fin tienen tantos JSON y despues decidimos que hacemos con ellos, ?que dicen @edoomm  y @JavierMung ?

edoomm commented:
va, es que igual pongo peros, porque no sé cuánto pesen ¿? jajaja tenemos límite aquí en el Github, no me gustaría que el repo empiece a chillar porque andamos excediendo el límite de algo que no estamos usando por el momento o que nos puede abarcar mucho espacio en nuestras PCs

MarcoOrtegaF commented:
Según yo no son tan grandes, no veo más de 200 trayectorias en el archivo más grande, pero mejor lo revisamos en la reunión de hoy y ya vemos que hacer con esos assets mientras dejamos la pull abierta

edoomm commented on assets/GEOJSON 2/Atizapan.json:
este que pedo

edoomm commented on utils/plotfn.py:
```diff
@@ -133,3 +133,6 @@ def time_complexity(
 
     plt.show()
+    
+def test():
```
quita esta funcion

MarcoOrtegaF committed:
Update plotfn.py

Eliminada la funcion de prueba

MarcoOrtegaF committed:
Update Atizapan.json

Ahora tiene las trayectorias correspondientes

MarcoOrtegaF committed:
Lo mismo pero sin funciones en C++ para que Ruff no se ponga mamon

MarcoOrtegaF committed:
Merge branch 'MarcoOrtegaF/issue29' of https://github.com/TT2024-B106/similarity-measures into MarcoOrtegaF/issue29

MarcoOrtegaF committed:
Correccion de la celda que tenia conflicto con Ruff

edoomm commented:
Acá esta tu pedo
![image](https://github.com/user-attachments/assets/e8f4bd2a-8353-4a5c-98a8-a3a50afbf5fc)

Que es en tu jupyter, andas importando 2 veces `json` por ejemplo en la primera chilladera del ruff
![image](https://github.com/user-attachments/assets/4bce5e0c-2d04-4e5f-a75a-d364707dcb98)
![image](https://github.com/user-attachments/assets/e7966c60-f113-43f4-8ec8-b5aa96d2bb1d)

Chécale bien los errores que te anda arrojando mi todo tibio

MarcoOrtegaF committed:
Correcciones a los noteboooks, se eliminaron librerias que estaban duplicadas

MarcoOrtegaF commented:
Ya corregi los errores de los Notebooks mi tio @edoomm, en los tuyos tambien habia problemas con las librerias, tambien ya estan corregidos

MarcoOrtegaF committed:
Borre los txt, Yo creo que con los JSON esta bien

edoomm reviewed (approved):
Te faltó meter tu función para graficar en `plotfn.py` y te sobra una celda de código en tu notebook, pero ya la apruebo para que no chillen las otras PR
